### PR TITLE
feat: api config services

### PR DESCRIPTION
**Pull request by HAPPYS1NGH**
Added Backend and APIs for config

### CODE REVIEW

File: apps/app/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.env
+/node_modules/
```
Comment (Markkos89) [inline apps/app/.gitignore]: this `.gitignore` file is not needd. Can you delete it? thanks!

File: apps/app/app.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+// Middleware
+app.use(bodyParser.json());
+
+// Define Routes (to be created in the next step)
+const configRoutes = require("./routes/config");
+app.use("/api/config", configRoutes);
+
+app.listen(port, () => {
+  console.log(`Server is running on port ${port}`);
+});
```
Comment (Markkos89) [inline apps/app/app.js]: can you change the name of the app's folder?

name it instead of `app` --- as ---> `config-api` :pray: 

thank you in advance!

**Review by Markkos89 — CHANGES_REQUESTED** [review]
great work, thank you @HAPPYS1NGH 
 
 two small changes if you can address them :pray:

File: apps/app/app.js (hunk 2 above)
Comment (HAPPYS1NGH) [reply apps/app/app.js]: I have done those changes

**Review by Markkos89 — APPROVED** [review]

**Pull request merged**